packages/web-frontend/src/selectors/measureSelectors.js

```diff
@@ -136,7 +136,7 @@ export const selectMeasuresWithDisplayInfo = createSelector(
     hiddenMeasures,
     currentCountry,
   ) => {
-    if (!currentCountry || !measureData || !country) {
```

Comment:
That's where the bug I mentioned above comes from. We don't actually need to check that with [];